library/aur.py

```diff
@@ -102,6 +111,11 @@
     'makepkg': ['makepkg', '--syncdeps', '--install', '--noconfirm', '--needed']
 }
 
+use_cmd_local_pkgbuild = {
+    'pikaur': ['pikaur', '-P', '--noconfirm', '--noedit', '--needed', '--install'],
+    'makepkg': ['makepkg', '--syncdeps', '--rmdeps', '--install', '--noconfirm', '--needed']
```

Comment:
I would remove `--rmdeps` here, it can be added with extra_args if needed
```suggestion
    'makepkg': ['makepkg', '--syncdeps', '--install', '--noconfirm', '--needed']
```